auth/realm/base/src/main/resources/schema/elytron-identity-1_2.xsd

```diff
@@ -21,18 +21,22 @@
 <xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
             targetNamespace="urn:elytron:identity:1.2"
             xmlns="urn:elytron:identity:1.2"
+            xmlns:xmldsig="http://www.w3.org/2000/09/xmldsig#"
             elementFormDefault="qualified"
             attributeFormDefault="unqualified"
             version="1.2">
 
     <!-- File-backed realm elements -->
 
     <xsd:element name="identity" type="identity-type"/>
+    <xsd:import namespace="http://www.w3.org/2000/09/xmldsig#" schemaLocation="https://www.w3.org/TR/2002/REC-xmldsig-core-20020212/xmldsig-core-schema.xsd" />
 
     <xsd:complexType name="identity-type">
         <xsd:all minOccurs="1" maxOccurs="1">
+            <xsd:element name="principal" type="name-type" minOccurs="1" maxOccurs="1"/>
             <xsd:element name="credentials" type="credentials-type" minOccurs="0" maxOccurs="1"/>
             <xsd:element name="attributes" type="attributes-type" minOccurs="0" maxOccurs="1"/>
+            <xsd:element name="Signature" />
```

Comment:
A `type` attribute should be added here that references the `SignatureType` from the xml dsig schema.

Reply:
Should set `minOccurs` and `maxOccurs` too.

Reply:
Updated